x-pack/legacy/plugins/fleet/public/components/agent_health.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiHealth } from '@elastic/eui';
+
+interface Props {
+  agent: any;
```

Comment:
I believe this type should be [`Agent`](https://github.com/elastic/kibana/blob/feature-make-it-ingest/x-pack/legacy/plugins/fleet/common/types/domain_data.ts#L33) but the response doesn't return that type atm, so just using `any` for now

Reply:
it should be Agent less access_token and enrollment_token... we prob need to adjust these types a bit / create a new type

Reply:
Yes, I re adjusted the type for the agent detail page, I can create a PR with it later tonight